R/manifest-convert.R

```diff
@@ -20,18 +21,11 @@
 #'   object; otherwise, the lockfile will be written to the path specified by
 #'   `lockfile`.
 #'
-#' @details
```

Comment:
This appeared to say the same thing as the parameter documentation, so I deleted it.

I do wonder if `renv.lock` should be the default, so you can then call `renv::restore()` to get the packages you need.